README.md

```diff
@@ -15,6 +15,96 @@ gradient-free optimizers, and constrained optimization.
 * Armadillo: http://arma.sourceforge.net
 * OpenBLAS or Intel MKL or LAPACK (see Armadillo site for details)
 
+### Install Dependencies
+
+<details open>
+<summary>Linux</summary>
+
+Use your distributions' package manager to install the required dependencies. The commands shown below are for Ubuntu.
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install -y g++ cmake
+$ sudo apt-get install -y libopenblas-dev liblapack-dev xz-utils
```

Comment:
No need to install those packages, armadillo is already packaged with a BLAS variant, and xz utils are only necessary to extract the armadillo archive, which isn't needed if you install it via the package manager.

Reply:
Okay, will remove these.